docs/en-US/commands/Find-JiraFilter.md

```diff
@@ -0,0 +1,298 @@
+---
+external help file: JiraPS-help.xml
+Module Name: JiraPS
+online version: https://atlassianps.org/docs/JiraPS/commands/Find-JiraFilter/
+locale: en-US
+schema: 2.0.0
+layout: documentation
+permalink: /docs/JiraPS/commands/Find-JiraFilter/
+---
+# Find-JiraFilter
+
+## SYNOPSIS
+
+Find JIRA filter(s).
+
+## SYNTAX
+
+### ByInputObject
+
+```powershell
+Find-JiraFilter [[-Name] <string[]>] [[-AccountId] <string>] [[-GroupName] <string>] [[-Project] <Object>] [[-Fields] {description | favourite | favouritedCount | jql | owner |
+    searchUrl | sharePermissions | subscriptions | viewUrl}] [[-Sort] <string>] [[-Credential] <pscredential>] [-IncludeTotalCount] [-Skip <uint64>] [-First <uint64>]
+    [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Searches for filters. This operation is similar to Get filters except that the results can be refined to include filters that have specific attributes. For example, filters with a particular name. When multiple attributes are specified only filters matching all attributes are returned.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Find-JiraFilter -Name 'ABC'
+```
+
+This example finds all JIRA filters that include ABC in the name.  The search is case insensitive.
+
+### EXAMPLE 2
+
+```powershell
+Find-JiraFilter -Name """George Jetsons Filter"""
+```
+
+This example finds a JIRA filter by exact name (case insensitive)
+
+### EXAMPLE 3
+
+```powershell
+'My','Your' | Find-JiraFilter
+```
+
+This example demonstrates use of the pipeline to search for multiple filter Name(s).  The search is case insensitive.
+
+### EXAMPLE 4
+
+```powershell
+Find-JiraFilter -Name 'My','Your'
+```
+
+This example demonstrates the use of a list of names to search for multiple filter Name(s).  The search is case insensitive.
+
+### EXAMPLE 5
+
+```powershell
+Find-JiraFilter -AccountId 'owner' -First 3 -Skip 3
+```
+
+This example finds all JIRA filters belonging to a specific owner, and illustrates the use of the -First and -Skip Paging parameters.
+
+### EXAMPLE 6
+
+```powershell
+Find-JiraFilter -Project 10315 -First 8
```

Comment:
I would prefer an example with the Project as a key

Reply:
done.